.github/workflows/rust.yml

```diff
@@ -321,6 +321,14 @@ jobs:
           destination: "rerun-web-viewer/version/${{github.ref_name}}"
           parent: false
 
+      - name: "Upload web-viewer (latest release)"
+        if: github.ref == 'latest'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "web_viewer"
+          destination: "rerun-web-viewer/web_viewer"
```

Comment:
I'm no longer sure this is right.

Where does <https://app.rerun.io/> even serve from? O.o

Reply:
We’ll want to modify the base rewrite rule to send app.rerun.io to whatever prefix we use for this target.